site/content/entry/citra-progress-report-2018-Q1.md

```diff
@@ -0,0 +1,230 @@
+
+One of the highly requested features of Citra has been the support for using the system camera to
+scan QR codes, etc. [Zhaowenlan1779](https://github.com/zhaowenlan1779) took it upon himself to
+provide Citra with this feature. By making use of the available Qt libraries, he worked on the
+implementation for weeks, and finally, with collaboration from other contributors, he was able to
+finish the feature. Citra now boasts its ability to support scanning images or using the system
+camera to function in a manner similar to the 3DS camera.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q1/camera.jpg"
+    title="System Camera Support" >}}
+
+## [citra-qt: Translations](https://github.com/citra-emu/citra/pull/3297) by [wwylele](https://github.com/wwylele) and [The Citra Community](https://community.citra-emu.org)
+
+For a long while now, Citra’s UI had been released only in English and had absolutely no support for
+other languages. [wwylele](https://github.com/wwylele) discussed this at length with other contributors
+and finally brought UI translations to Citra. Citra can now be used in many other languages. Note that
+these translations are maintained by our own community members and the translations are hosted on
+[Transifex](https://www.transifex.com/citra/citra/). You too, can submit a translation for your language
+(if it doesn’t exist already), or review existing translations to make them accurate.
+
+{{< figure src="/images/entry/citra-progress-report-2018-q1/translation.jpeg"
+    title="Citra - Now available in your language" >}}
+
+## [Sending Test Cases via Telemetry](https://github.com/citra-emu/citra/pull/3325) by [BreadFish64](https://github.com/BreadFish64)
+
+Over the years, Citra has improved by leaps and bounds. However, there has always been a lack of consistent
+information on game compatibility. We introduced the [game compatibility list](https://citra-emu.org/game/)
+to fix that. Having a list was helpful, but maintaining it was proving to be a tedious task. After a
+lot of brainstorming, [BreadFish64](https://github.com/BreadFish64) added the option in the UI to send
+test cases via telemetry. These results are submitted by the users themselves, and are in turn used to
+update the game compatibility list.
+
+### Steps to submit test cases via Telemetry
+{{< sidebyside "image" "/images/entry/citra-progress-report-2018-q1/"
+    "compat-1.png=Step 1"
+    "compat-2.png=Step 2" >}}
+{{< sidebyside "image" "/images/entry/citra-progress-report-2018-q1/"
+    "compat-3.png=Step 3"
+    "compat-4.png=Step 4" >}}
+**Note**: To be able to report compatibility, you have to first login to the Citra Web Service, using
+your Username and Token.</br> (See Emulation -> Configure -> Web)
+{{< figure src="/images/entry/citra-progress-report-2018-q1/web-verify.png"
+    title="Citra Web Service" >}}
+&nbsp;
```

Comment:
May I ask why the `br` tag and non-breaking space? Although raw HTML is allowed in Markdown, it's preferable that instead native Markdown or hugo constructs are used. It's what motivated @chris062689 to make a custom hugo macro for the 2017 September report instead of a horrible HTML hack I was using for the draft.

Reply:
I wasn't aware of flame sage's efforts. I previously worked on static html pages only. I used "<br>" to send that (See....) down to next line. I used nbsp to create some space between pics and next header